bindings/nodejs/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![deny(clippy::all)]
+
+#[macro_use]
+extern crate serde;
```

Comment:
Nit: we no longer need `extern crate` since the 2018 edition. Should also forbid unsafe code.

```suggestion
#![forbid(unsafe_code)]
#![deny(clippy::all)]
```